ledger/ledger-runner-common/src/main/scala/com/daml/ledger/runner/common/PureConfigReaderWriter.scala

```diff
@@ -228,6 +238,9 @@ class PureConfigReaderWriter(secure: Boolean = true) {
 
   implicit val seedingWriter: ConfigWriter[Seeding] = ConfigWriter.toString(_.name)
 
+  implicit val userManagementConfigHint =
+    ProductHint[UserManagementConfig](allowUnknownKeys = false)
+
```

Comment:
Seems `jwtTimestampLeewayConfigConvert` below is missing a hint

Reply:
ah actually all optional configs, let me fix that

Reply:
done